pages/linux/deluser.md

```diff
@@ -0,0 +1,20 @@
+# deluser
+
+> Delete user from system.
+> All commands must be executed as root (`sudo [command]`).
+
+- Remove a user:
+
+`deluser {{name}}`
```

Comment:
A user might be confused and think display name, e.g. so they might run `deluser John Doe` instead of `deluser johndoe` ;)
```suggestion
`deluser {{username}}`
```